InriaMuseum/InriaMuseum.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Inria Museum</title>
+    <link rel="stylesheet" type="text/css" href="InriaMuseum.css"> 
+    <script>
+
+
+        var current_page; //global variable
+        var previous_page;
+        function handleButtonClick(event) { //event listener for buttons
+            console.log(`ct handleButtonClick ${event}`);
+            console.log(`event.target.id ${event.target.id}`);
+
+            if(event.target.id == ""){
+                const parentElementId = event.target.parentElement ?
+                event.target.parentElement.id : 'No parent element';
+                console.log(`event.target.id ${event.target.parentElement.id}`)
+                changePage(parentElementId);
+            }else{
+                changePage(event.target.id);
+            }
+        }
+        
+        function changePage(ElementId) {
```

Comment:
I believe that in Javascript, `ElementId` is a reserved word, which you are overloading here. Please rename to `parent_element`